app/views/pages/faq.html.haml

```diff
@@ -1,23 +1,43 @@
 - content_for :page_title do
-  Fequently Asked Questions and Other Things
+  Frequently Asked Questions and Other Things
 
 %h1.dividing.header FAQ
 
 %h2.dividing.header General
+%p
+  %b.question Who are you?
+  .answer
+    I'm just a peep in search of that perfect bookmark.
+
+%p
+  %b.question What happens if you're ran over?
+  .answer
+    They tend to be a lot larger than me, so the car/bus/truck is probably going to need to clean their windshield and scrape an arm or leg off of their side mirror. Eventually the server funds will run out and transientBug will be shutdown by some automated billing process which will proceed to gleefully pat itself on the back and continue on with its harrowing existence.
+
 %p
   %b.question Can I make suggestions for new features?
   .answer
-    Yup, send an email to suggestions (at) transientbug.ninja and I'll look into it.
+    Yup, you can either make an #{ link_to "issue", "https://github.com/transientBug/transientbug-rails/issues" } with the "suggestion" tag, or send an email to suggestions (at) transientbug.ninja and I'll look into it.
 
 %p
   %b.question How can I report a bug?
   .answer
-    You should send an email to issues (at) transientbug.ninja with any information you can. The more information I've got, the easier it is to track down the bug; Bonus points: I'll let you know when the bug is fixed, or which github issue or trello card you can use to track the progress.
+    %p
+      If it is a bug with the website, you should use #{ link_to "these Github issues", "https://github.com/transientBug/transientbug-rails/issues" }.
+
+    %p
+      If the bug is a part of the Firefox extension, you should use #{ link_to "these Github issues", "https://github.com/transientBug/firefox-extension/issues" } instead.
+
+    %p
+      For any issue: please make sure that you aren't making a duplicate of an existing report. Also, please include as much information as you can and be detailed in the actions leading to the bug. The more information I've got, the easier it is to track down the bug; Bonus points: I'll let you know when the bug is fixed, or which github pull request or trello card you can use to track the progress.
+
+    %p
+      Alternatively, you can send an email to issues (at) transientbug.ninja and I can put it into a Github issue.
```

Comment:
- ~Alternatively you can send~ `You can also send`
- ~I can put it into a Github issue~